src/Caching/Hybrid/test/ServiceConstructionTests.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Caching.Hybrid.Internal;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.DependencyInjection;
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+#pragma warning disable CS8769 // Nullability of reference types in type of parameter doesn't match implemented member (possibly because of nullability attributes).
```

Comment:
Can we avoid both of these?

Reply:
the latter is a consequence of IVT and a type conflict if I try to match; the former is just for the dummy trivial async callbacks - I can *not* do that, but it bends the test out of shape; neither is particularly worth fighting, IMO